tests/helpers.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+"""Test helpers."""
+
+
+def simple_check(data):
+    """A check that returns a warning."""
+    errors = {}
+    if data.get('hello', '') == 'world':
+        errors['hello'] = [{
```

Comment:
I am assuming that checker functions will return in the `key` that has the problem a list of errors (even if there is only one). So after I use `extend` - https://github.com/inveniosoftware-contrib/invenio-record-editor/pull/12/files#diff-71a4f12cd34d737a14762152409ffef7R66

I am not sure if we should allow to return also just a `dict` for cases with only one problem and deal with it on `RecordValidator.validate`